common/src/main/java/io/seata/common/loader/EnhancedServiceLoader.java

```diff
@@ -84,7 +77,8 @@
      * @throws EnhancedServiceNotFoundException the enhanced service not found exception
      */
     public static <S> S load(Class<S> service, String activateName) throws EnhancedServiceNotFoundException {
-        return loadFile(service, activateName, findClassLoader());
+        return InnerEnhancedServiceLoader.getServiceLoader(service).load(activateName,
```

Comment:
It is recommended that findClassLoader() be placed in the EnhancedServiceLoader class, which is responsible for the parameter assembly.

Reply:
OK,now i place it in the EnhancedServiceLoader, and uniformly construct the parameters before calling the inner method.